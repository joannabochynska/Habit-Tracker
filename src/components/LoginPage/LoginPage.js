import React, { Component } from 'react';
import styles from './LoginPage.module.scss';

class LoginPage extends Component {
  render() {
    return (
      <div className={styles.login}>
        <button className={styles.facebook}>
            Log in with Facebook
        </button>
        <button className={styles.github}>
            Log in with GitHub
        </button>
      </div>
    );
  }
}

export default LoginPage;
