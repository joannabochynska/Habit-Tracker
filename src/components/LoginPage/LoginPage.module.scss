@import "../../scss/constants";

.login {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding-top: 40px;
    padding-bottom: 40px;
    height: 200px;
    align-content: space-around;

    button {
        height: 50px;
        width: 400px;
        margin-top: 30px;
    }
}