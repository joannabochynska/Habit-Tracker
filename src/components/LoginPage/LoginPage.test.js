import React from 'react';
import renderer from 'react-test-renderer';
import LoginPage from './LoginPage';

test('renders correctly', () => {
  const tree = renderer
    .create(<LoginPage />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
