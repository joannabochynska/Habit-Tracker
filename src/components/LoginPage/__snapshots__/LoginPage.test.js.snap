// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="login"
>
  <button
    className="facebook"
  >
    Log in with Facebook
  </button>
  <button
    className="github"
  >
    Log in with GitHub
  </button>
</div>
`;
